### ISSUE TITLE
‌‌‌‌‌After using the Huggingface integration solution, will a model occupy twice the disk space?

### ISSUE THREAD

**Issue by yetone** [dragonflyoss/dragonfly#3270]
‌‌‌‌‌‌I'm very happy to see the integration solution of Huggingface. I'm curious whether this will cause Hugging Face's models to occupy double disk space (dragonfly cache directory + huggingface cache directory)?

https://d7y.io/docs/next/operations/integrations/hugging-face/

**Comment by gaius-qi** [comment]
Yes, it is possible to optimize Dfdaemon's GC configuration and reduce the cache size .
Refer to https://d7y.io/docs/next/reference/configuration/dfdaemon.

![image](https://github.com/dragonflyoss/Dragonfly2/assets/15955374/9732e9cf-fd8a-4fd7-a561-99076e13ea88)

**Comment by yetone** [comment]
‌‌‌Thanks, I'll look into it!

**Comment by yetone** [comment]
‌‌‌‌Will there be any adverse effects if I completely use dragonfly's cache and disable hugging face's own cache? Specifically, our goal is to have different containers utilize the same model cache.

**Comment by gaius-qi** [comment]
> @gaius-qi Will there be any adverse effects if I completely use dragonfly's cache and disable hugging face's own cache? Specifically, our goal is to have different containers utilize the same model cache.

If model has cached in d7y, huggingface sdk will download by http proxy. Slower but reduce the space.
